rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -2064,7 +2064,8 @@ public ResponseEntity<?> importSBOMonProject(
             @Parameter(description = "Project ID", example = "376576")
             @PathVariable(value = "id", required = true) String id,
             @Parameter(description = "SBOM file")
-            @RequestBody MultipartFile file
+            @RequestBody MultipartFile file,
+            @RequestParam(value = "doNotReplacePackageAndRelease", required = false) boolean doNotReplacePackageAndRelease
```

Comment:
Please add OpenAPI doc for the parameter with `@Parameter` annotation.

Reply:
@GMishx I have made the asked change. Please review.

Reply:
Would recommend a simpler description like "Overwrite existing project releases and packages while importing new SBOM?"

Reply:
@GMishx Please review.